### ISSUE TITLE
Adding Terms of Use to Application Pages

### ISSUE THREAD

**Issue by craigspaeth** [artsy/force#1406]
Request to add in our Terms of Use to the following pages under the form: 

![screen shot 2017-04-11 at 5 11 35 pm](https://cloud.githubusercontent.com/assets/6314270/25134370/5dc47e08-241d-11e7-995e-b58411f55dba.png)

1. Artsy.net pages
- https://www.artsy.net/gallery-partnerships
- https://www.artsy.net/auction-partnerships/support

2. Marketo lps 
- http://pages.artsy.net/gallery-partnerships
- http://pages.artsy.net/galleries.html

Which use the following templates - 
https://app-ab14.marketo.com/#LT1066B2ZN12
https://app-ab14.marketo.com/#LT1086B2ZN12

Once the templates are adjusted, we can then apply the changes to remaining marketo application pages like  http://pages.artsy.net/general.html, http://pages.artsy.net/fairs.html etc

cc @sperle

**Comment by craigspaeth** [comment]
@cfinsness

**Comment by cfinsness** [comment]
Hey, after we edited the marketo form the TOC appeared twice on https://www.artsy.net/gallery-partnerships/apply

<img width="761" alt="screen shot 2017-05-11 at 10 18 34 am" src="https://cloud.githubusercontent.com/assets/6314270/25954369/16ff07ee-3634-11e7-97b5-35da5e5d324a.png">

I've removed the text from the marketo form itself for now. But I see two options - (?)
1. Keeping the text on the marketo form itself and removing the text under the button here
2. Keeping the text under the button (+ adding Terms of Use) and using a separate form for the landing pages hosted on marketo. 

lmk what you think @craigspaeth
